scripts/Tools/case.submit

```diff
@@ -115,7 +116,12 @@ sub main
 	    check_DA_settings();
 	    `./xmlchange --noecho  RUN_WITH_SUBMIT=TRUE`;
 	    $config{RUN_WITH_SUBMIT}='TRUE';
-	    $batchutils->submitJobs();
+            if (defined $opts{'jobid'}) {
```

Comment:
I'm about to remove this script.  I think maybe someone was trying to improve the flawed logic here.   Is there a log note or an issue explaining what this was to address?